WEB_SERVICE/WEB/_SECURE/SECURE_HTTP_HEADER/description.md

```diff
@@ -0,0 +1,12 @@
+Secure Header Setting:
```

Comment:
This could benefit from a smapp introduction describing http header settings.

Reply:
+1